### ISSUE TITLE
Test failling

### ISSUE THREAD

**Issue by ph** [logstash-plugins/logstash-filter-metrics#21]
```
  1) LogStash::Filters::Metrics with multiple instances counts should be independent
     Failure/Error: insist { events_tag1.first["http.200.count"] } == 1
     Insist::Failure:
       Expected 1, but got 3
     # ./spec/filters/metrics_spec.rb:107:in `(root)'
```

**Comment by hp197** [comment]
Saw it yesterday as well, but was paying more attention to my other pull.

**Comment by jsvd** [comment]
fixed by #22, new version of the plugin has been published
